set-web/src/main/java/eu/europeana/set/web/service/controller/jsonld/WebUserSetRest.java

```diff
@@ -119,7 +125,8 @@
 			WebUserSetFields.MAX_ITEMS_PER_PAGE, profile);
 	    }
 
-	    String serializedUserSetJsonLdStr = serializeUserSet(profile, storedUserSet);
+	    UserSet resUserSet = setTotalField(profile, storedUserSet);
```

Comment:
acording to the desing this functionality must be implemented in the  applyProfile method which is already called in serializeUserSet method.
UserSet set = getUserSetService().applyProfile(storedUserSet, profile);

Reply:
functionality moved to applyProfile method